pkg/ext/stores/tokens/tokens.go

```diff
@@ -625,7 +625,9 @@ func (t *SystemStore) list(fullAccess bool, userName, sessionID string, options
 
 	return &ext.TokenList{
 		ListMeta: metav1.ListMeta{
-			ResourceVersion: secrets.ResourceVersion,
+			ResourceVersion:    secrets.ResourceVersion,
+			Continue:           secrets.Continue,
+			RemainingItemCount: secrets.RemainingItemCount,
```

Comment:
Note: this can be affected by L618

Reply:
Agreed. No quick idea how handle. Or if we even should.